integration/spark2/src/main/java/org/apache/carbondata/spark/vectorreader/VectorizedCarbonRecordReader.java

```diff
@@ -134,9 +133,7 @@ public void initialize(InputSplit inputSplit, TaskAttemptContext taskAttemptCont
     queryModel.setTableBlockInfos(tableBlockInfoList);
     queryModel.setVectorReader(true);
     try {
-      if (CarbonProperties.getInstance().getProperty(
-              CarbonCommonConstants.CARBON_SEARCH_MODE_ENABLE,
-              CarbonCommonConstants.CARBON_SEARCH_MODE_ENABLE_DEFAULT).equals("true")) {
+      if (CarbonProperties.isSearchModeEnabled()) {
```

Comment:
I guess no need to do if check here, already `QueryExecutorFactory` available so move the code there.

Reply:
fixed